salt/orch/update-etc-hosts.sls

```diff
@@ -1,15 +1,20 @@
+update_pillar:
+  salt.function:
+    - tgt: '*'
+    - name: saltutil.refresh_pillar
+
 update_grains:
   salt.function:
-    - tgt: 'roles:kube-(master|minion)'
-    - tgt_type: grain_pcre
-    - name: saltutil.sync_all
+    - tgt: '*'
```

Comment:
Please double check this `'*'` works fine, because I remember it dumped some errors on the `ca`.